tests/Behat/Context/UI/ManagingAdministrationRolesContext.php

```diff
@@ -56,6 +59,33 @@ public function wantToAddSomePermissionsToAdministrationRole(AdministrationRoleI
         $this->updatePage->open(['id' => $administrationRole->getId()]);
     }
 
+    /**
+     * @When I want to edit a non existent administration role
+     */
+    public function wantToEditNonExistentAdministrationRole(): void
+    {
+        try {
+            $this->updatePage->open(['id' => 999]);
+        } catch (UnexpectedPageException $exception) {
```

Comment:
Such an exception will not be thrown, as de driver would follow the redirect. `$this->updatePage->open(['id' => 999]);` is eveything that we need here